### ISSUE TITLE
🚀 Enhancement: Add  `Query.deselect` to Queries

### ISSUE THREAD

**Issue by soranoo** [appwrite/appwrite#8380]
### 🔖 Enhancement description

`Query.select` enables us to select the fields we need but how about we get a lot of fields and we want to exclude a field? There is no handy way to do so.

### 🎤 Pitch

`Query.deselect` should do the opposite of `Query.select` do.

> Query.deselect(["name", "title"])

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by neolizee** [comment]
This is a good offer! If you need to exclude only 2 out of 20 parameters in the response, instead of listing all the necessary ones. 🌟

**Comment by byawitz** [comment]
Hey @soranoo, thanks for opening the issue, good idea.

In the SQL and NoSQL world getting data always goes the `select` way, when you're selecting what you want.
Adding `deselect` will make every request twice as long, as the only way to deselect attributes is by know first what attributes are to select.

**Comment by neolizee** [comment]
In SQL there is SELECT * EXCEPT(column_to_exclude)

**Comment by byawitz** [comment]
Are you sure this is a valid MySQL/MariaDB syntax?

It seems like it's a different command
- https://mariadb.com/kb/en/except/
- https://dev.mysql.com/doc/refman/8.4/en/except.html

**Comment by neolizee** [comment]
Used in Google BigQuery

**Comment by byawitz** [comment]
It's a different database and doesn't support Appwrite [utopia-php/database](https://github.com/utopia-php/database/tree/main/src/Database/Adapter)

**Comment by neolizee** [comment]
That's right, I just pointed out that there is such a cool thing. And it would be cool if this appeared in AW.

**Comment by soranoo** [comment]
> Hey @soranoo, thanks for opening the issue, good idea.
> 
> In the SQL and NoSQL world getting data always goes the `select` way, when you're selecting what you want. Adding `deselect` will make every request twice as long, as the only way to deselect attributes is by know first what attributes are to select.

Is it possible to forcibly cache all attributes to ram or redis? so we can access them as fast as possible. I believe they won't take up too much space. After that, we can write down performance tradeoffs inside the docs.

**Comment by Jacobojijo** [comment]
Hello @soranoo @eldadfux 
Kindly assign me this issue
